lua/proto/tcp.lua

```diff
@@ -470,11 +489,192 @@ function tcpHeader:getUrgentPointerString()
 	return self:getUrgentPointer()
 end
 
--- TODO how do we want to handle options (problem is tcp header variable length array of uint8[] followed by payload variable length array (uint8[]))
---[[function tcpHeader:setOptions(int)
-	int = int or
-	self. = int
-end--]]
+
+------------------------------------------------------------------------------------
+---- TCP options
+------------------------------------------------------------------------------------
+
+do 
+	function createOptionsDict()
+		mod.option = {
+			err = -1,
+			eol = 0,
+			nop = 1,
+			mss = 2,
+			ws = 3,
+			ts = 8,
+		}
+		for k, v in pairs(mod.option) do
+			mod.option[v] = k
+		end
+	end
+	createOptionsDict()
```

Comment:
seems pointless to declare a function and then call it immediately. just get rid of the function definition and call?